spec/index.html

```diff
@@ -733,125 +733,140 @@ <h3>Literals</h3>
     <p>Literals are used for values such as strings, numbers, and dates.</p>
 
     <p>A <dfn data-local-lt="RDF literal">literal</dfn> in an <a>RDF graph</a> consists of
-      two, three, or four elements, as follow:</p>
+      two, three, or four elements, as follow.</p>
```

Comment:
This was intentionally a colon. A full-stop puts a bit too much break.
```suggestion
      two, three, or four elements, as follow:</p>
```

Reply:
Since the bullets of the list have been converted to full sentences (starting with a capital letter, ended with a period), it seemed more grammatically correct to mark the end of a sentence here. But if you confirm that this is ok, I defer to your better knowledge of English grammar.

Reply:
The colon takes the place of the full-stop. It is correct English punctuation.